pulpcore/pulpcore/app/tasks/base.py

```diff
@@ -0,0 +1,56 @@
+from celery import shared_task
```

Comment:
Both pulpcore/pulpcore/app/tasks/base.py and platform/pulpcore/app/tasks/base.py?

Reply:
No .... that was wrong on my part. This got put in as part of the rebase :(